backend/exchanges/moonpay.go

```diff
@@ -36,7 +36,16 @@ var moonpayAPICryptoCode = map[coin.Code]string{ // -> moonpay crypto currency c
 	coin.CodeBTC: "btc",
 	coin.CodeLTC: "ltc",
 	coin.CodeETH: "eth",
-	// TODO: ERC20 tokens; figure out eth-erc20- prefix vs frontend
+
+	"eth-erc20-usdt":      "usdt",
+	"eth-erc20-usdc":      "usdc",
+	"eth-erc20-bat":       "bat",
+	"eth-erc20-dai0x6b17": "dai",
```

Comment:
What about `eth-erc20-sai0x89d2` or is it unsupported?

Reply:
yes, unsupported. afaik it's the predecessor of DAI.